base/services/w32time/w32time.rc

```diff
@@ -42,3 +42,6 @@
 #ifdef LANGUAGE_ZH_TW
     #include "lang/zh-TW.rc"
 #endif
+#ifdef LANGUAGE_IT_IT
+    #include "lang/it-IT.rc"
+#endif
```

Comment:
Should be added in alphabetical order, not to the end of file. (same for the other one)